src/actions/actions.js

```diff
@@ -0,0 +1,27 @@
+export const chooseFilm = (film) => ({
+  type: "FILM_SELECTED",
+  payload: film,
+});
+
+export const fetchedDataSuccess = (films) => ({
+  type: "FETCH_DATA",
+  payload: films,
+});
+
+export const sortByDate = (films, sortBy) => ({
+  type: "SORT_BY_DATE",
+  payload: films,
+  sortBy: sortBy
```

Comment:
'sortBy' is not understandable name. Please rename it and move to payload.

Reply:
done